coop_membership/data/email_template_data.xml

```diff
@@ -8,7 +8,47 @@
         <field name="lang">${object.lang}</field>
         <field name="reply_to">${(object.company_id.email or '')|safe}</field>
         <field name="subject">Welcome to Coop</field>
-        <field name="body_html"><![CDATA[]]></field>
+        <field name="body_html"><![CDATA[
+<p>Hello ${object.name},</p>
+
+<p>Your subscription to La Louve has been registered. Welcome !</p>
```

Comment:
"La Louve" should be replaced by object.company_id.company_name

Reply:
@ivantodorovich : done